File: Cpp/zerojudge/基礎題庫/a006.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
	int a, b, c, ans;
	while (cin >> a >> b >> c)
	{
		if ((b * b - 4 * a * c) < 0)
			cout << "No real root" << endl;
		else if ((b * b - 4 * a * c) == 0)
			cout << "Two same roots x=" << (-b + sqrt(b * b - 4 * a * c)) / (2 * a) << endl;
		else
			cout << "Two different roots x1=" << (-b + sqrt(b * b - 4 * a * c)) / (2 * a) << " , x2=" << (-b - sqrt(b * b - 4 * a * c)) / (2 * a) << endl;
	}
}

File: Python/zerojudge/a006.py
from math import sqrt
s=str(input())
a=int(s.split()[0])
b=int(s.split()[1])
c=int(s.split()[2])
if b*b-4*a*c > 0 :
    print('Two different roots x1=%d , x2=%d' % ((-b+sqrt(b**2-4*a*c))/(2*a),(-b-sqrt(b**2-4*a*c))/(2*a)))
elif b*b-4*a*c == 0 :
    print('Two same roots x=%d'%((-b+sqrt(b**2-4*a*c))/(2*a)))
else:
    print('No real root')